modules/ROOT/pages/upload-raml-task.adoc

```diff
@@ -7,17 +7,33 @@ You cannot publish an API specification to Anypoint Exchange without going throu
 
 == Before You Begin
 
-Within Anypoint Platform, your user ID must be assigned the _Design Center Developer_ permission.
+Within Anypoint Platform, your user ID must be assigned the Design Center Developer permission.
 
 == Procedure
 
-. In Anypoint Platform, open Design Center.
-. On the Projects page, click *Create* and select *API Specification*.
-. In the *New API Specification* dialog, specify a name for the new project, ensure that *Start with API Designer* is selected, and click *Create*.
-. In the project, click the three dots to the right of the heading *Files* and select *Import*.
-. In the *Import* dialog, select *File or Zip* and select the file from your computer, or select *URL* and specify the URL, if the file is online. Then, click *Import*.
-. Delete the file that API Designer created by default. This file has the same name as your project.
-. Click *Publish* > *Publish to Exchange*.
+. Open Design Center from the homepage of Anypoint Platform.
+. Create an API-specification project that you can work in.
+ .. On the *Projects* page in Design Center, click *Create* and then click *Create API Specification*.
+ .. In the *New Specification* section of the dialog that is opened, name your project. You can change the name later, if you want to.
```

Comment:
I have been editing for omitting locations, but use your best judgment.
The idea is that if the user is performing the task, the user sees the UI as controls open and close and can locate a UI label as long as it is not duplicated on a particular display.
The gear icon being located at the top is very common; even I would look there for this icon.
```suggestion
 .. In *New Specification*, name your project. 
 You can change the name later, if you want to.
```